machines/issuersMachine.ts

```diff
@@ -359,15 +359,22 @@ export const IssuersMachine = model.createMachine(
           onError: [
             {
               actions: [
-                log((_, event) => (event.data as Error).message),
+                log('Verification Error.'),
+                'resetLoadingReason',
+                'updateVerificationErrorMessage',
                 'sendErrorEndEvent',
               ],
               //TODO: Move to state according to the required flow when verification of VC fails
-              target: 'idle',
+              target: 'handlingCredentialVerificationFailure',
```

Comment:
can we rename the state to 'handleCredentialVerificationFailure' instead of 'handlingCredentialVerificationFailure'

Reply:
+1 to @vijay151096

Reply:
same here rename as `handleVCVerificationFailure`